i18n/fr-CH/fr_CH.po

```diff
@@ -3207,7 +3207,7 @@ msgstr "Gestion des modèles de rapports"
 
 #: app/scripts/reports/detail/reportsdetail.html:17
 msgid "Verändern"
-msgstr ""
+msgstr "Pauser"
```

Comment:
This is wrong, no?

Reply:
thanks, got inspired by the other languages. So, I assume I will need to change the others too...

Reply:
I guess.